docs/docs/atom-archive/getting-started/sections/atom-basics.md

```diff
@@ -0,0 +1,203 @@
+
+This is the Atom welcome screen and gives you a pretty good starting point for how to get started with the editor.
+
+#### Terminology
+
+You can find definitions for all of the various terms that we use throughout the manual in our [Glossary](/resources/sections/glossary/).
+
+#### Command Palette
+
+In that welcome screen, we are introduced to probably the most important command in Atom, the Command Palette. If you press <kbd class="platform-mac">Cmd+Shift+P</kbd><kbd class="platform-windows platform-linux">Ctrl+Shift+P</kbd> while focused in an editor pane, the command palette will pop up.
+
+::: tip Note:
+
+Throughout the book, we will use shortcut keybindings like <kbd class="platform-mac">Cmd+Shift+P</kbd><kbd class="platform-windows platform-linux">Ctrl+Shift+P</kbd> to demonstrate how to run a command. These are the default keybindings for the platform that we detected you running.
+
+If you want to see a different platform than the one we detected, you may choose a different one by using the platform selector near the top of the page:
+
+![Platform Selector](@images/atom/platform-selector.png "Platform Selector")
+
+If the Platform Selector is not present, then the current page doesn't have any platform-specific content.
+
+If you have customized your Atom keymap, you can always see the keybinding you have mapped in the Command Palette or the Keybindings tab in the [Settings View](#settings-and-preferences).
+
+:::
+
+This search-driven menu can do just about any major task that is possible in Atom. Instead of clicking around all the application menus to look for something, you can press <kbd class="platform-mac">Cmd+Shift+P</kbd><kbd class="platform-windows platform-linux">Ctrl+Shift+P</kbd> and search for the command.
+
+![Command Palette](@images/atom/command-palette.png "Command Palette")
+
+Not only can you see and quickly search through thousands of possible commands, but you can also see if there is a keybinding associated with it. This is great because it means you can guess your way to doing interesting things while also learning the shortcut key strokes for doing it.
+
+For the rest of the book, we will try to be clear as to the text you can search for in the Command Palette in addition to the keybinding for different commands.
+
+#### Settings and Preferences
+
+Atom has a number of settings and preferences you can modify in the Settings View.
+
+![Settings View](@images/atom/settings.png "Settings View")
+
+This includes things like changing the theme, specifying how to handle wrapping, font settings, tab size, scroll speed and much more. You can also use this screen to install new packages and themes, which we'll cover in [Atom Packages](/sections/atom-packages).
+
+To open the Settings View, you can:
+
+* Use the <span class="platform-mac">*Atom > Preferences*</span><span class="platform-windows">*File > Settings*</span><span class="platform-linux">*Edit > Preferences*</span> menu item in the menu bar
```

Comment:
Again, assigning HTML classes in Markdown should be avoided. Also, this is a horrible case of mixing HTML with Markdown formatting.